CL/cl_ext_intel.h

```diff
@@ -628,6 +629,24 @@ clEnqueueMemcpyINTEL_fn)(
             const cl_event* event_wait_list,
             cl_event* event);
 
+extern CL_API_ENTRY cl_mem CL_API_CALL
```

Comment:
Can you please distinguish these extension functions from the USM extension functions?  Right now they're intermingled with the USM extension functions, which is a little confusing.

Suggestions:

* Move these two functions to the end of this file.
* Add a header comment similar to the other extensions in this file:

```c
/*****************************************
* cl_intel_create_buffer_with_properties *
******************************************/
```